lib/AST/ASTVerifier.cpp

```diff
@@ -1816,15 +1816,6 @@ class Verifier : public ASTWalker {
         abort();
       }
       
-      // The base of a member reference cannot be an existential type.
-      if (E->getBase()->getType()->getWithoutSpecifierType()
```

Comment:
Maybe this should remain but isAnyExistentialType can be tightened up into isExistentialType (It’s still invalid to access a member type of an existential value)

Reply:
I noticed the same thing, was just adding that now, will have a commit up in a second.